plugin/config.go

```diff
@@ -18,37 +18,37 @@ type passwordConf struct {
 	MaxTTL int `json:"max_ttl"`
 
 	// Mutually exclusive with Length and Formatter
-	PolicyName string `json:"password_policy"`
+	PasswordPolicy string `json:"password_policy"`
 
-	// Length of the password to generate. Mutually exclusive with PolicyName.
+	// Length of the password to generate. Mutually exclusive with PasswordPolicy.
 	// Deprecated
 	Length int `json:"length"`
 
 	// Formatter describes how to format a password. This allows for prefixes and suffixes on the password.
-	// Mutually exclusive with PolicyName.
+	// Mutually exclusive with PasswordPolicy.
 	// Deprecated
 	Formatter string `json:"formatter"`
 }
 
 func (c passwordConf) Map() map[string]interface{} {
 	return map[string]interface{}{
-		"ttl":         c.TTL,
-		"max_ttl":     c.MaxTTL,
-		"length":      c.Length,
-		"formatter":   c.Formatter,
-		"policy_name": c.PolicyName,
```

Comment:
This feature is fairly new, but should we be worried that people might actually be expecting the policy name to be returned as "policy_name", but no longer shows once they upgrade?

Reply:
That's fair. @pcman312 and I chatted about this and we decided it probably isn't a big deal because:
* it's new
* it's a bug (you set it via one key and it returned another)
* it only shows up in a read of the config.

Do you think it's fair to still change this?

Reply:
Having those points taken into consideration, I think it's fine to change given the limited surface (only on config, fairly new) that it touches. It's also an unexpected field (and thus a bug) if the input and output parameters are expected to match but doesn't in this case.